src/site/content/en/blog/lowes/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Lowe's website is among fastest performing e-commerce websites"
+subhead: >
+  By building an automated performance testing and monitoring system,
+  the Lowe's Site Speed Team test pull requests against performance budgets,
+  and prevent performance regressions going into production.
+description: >
+  By building an automated performance testing and monitoring system,
+  the Lowe's Site Speed Team test pull requests against performance budgets,
+  and prevent performance regressions going into production.
+date: 2021-03-22
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/U0Xrfy2kVLEb3JSiPS5C.png
+alt: The Lowes logo.
+tags:
+  - blog
+  - case-study
+  - web-vitals
+  - performance
+---
+
+This post was authored by [Ashish Choudhury](https://www.linkedin.com/in/choudhuryashish/),
```

Comment:
We do still want to keep this note to highlight that the authors are from Lowe's.